src/services/sort.js

```diff
@@ -0,0 +1,59 @@
+import { saveToStorage, getTodos } from './update';
+
+export const handleDragStart = (evt) => {
+  evt.dataTransfer.setData('text/plain', evt.target.dataset.index);
+};
+
+export const sortList = () => {
+  const lists = document.querySelectorAll('.todoList li');
+  lists.forEach((list, index) => {
+    list.dataset.index = index;
+    list.querySelector('input').id = `todo-${index}`;
+    list.querySelector('label').setAttribute('for', `todo-${index}`);
+  });
+};
+
+export const cancelDefault = (e) => {
+  e.preventDefault();
+  e.stopPropagation();
+  return false;
+};
+
+export const handleDrop = (e) => {
+  cancelDefault(e);
+  const todos = getTodos();
+
+  // get new and old index
```

Comment:
- [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.